tutorials/plugins/editor/making_plugins.rst

```diff
@@ -72,6 +72,11 @@ EditorPlugin script for you. The script has two requirements that you cannot
 change: it must be a ``tool`` script, or else it will not load properly in the
 editor, and it must inherit from :ref:`class_EditorPlugin`.
 
+.. warning::
+    In addition to EditorPlugin script, any other GDScript that your plugin uses
```

Comment:
Here's a small correction, it should be good to go after this change:

```suggestion
    In addition to the EditorPlugin script, any other GDScript that your plugin uses
```

Reply:
Thanks, should be fixed!